### ISSUE TITLE
XKRX: FutureWarning: CustomBusinessDay.apply is deprecated ...

### ISSUE THREAD

**Issue by ValueRaider** [gerrymanoim/exchange_calendars#294]
```
import exchange_calendars as xcal
xcal.get_calendar("XKRX", start="2023-01-01", end="2024-01-01")
```

> exchange_calendars/pandas_extensions/offsets.py:152: FutureWarning: CustomBusinessDay.apply is deprecated and will be removed in a future version. Use `offset + other` instead

**Comment by maread99** [comment]
Yup. I seem to recall looking for this and it wasn't obvious to me what was causing it.

The 'XKRX' implementation is a pretty unique affair and there's been suggestions (#128)  to move all the holidays to hardcoded dates, which would almost certainly resolve this warning.

**Comment by maread99** [comment]
This will be fixed by #299.
